armi/reactor/tests/test_reactors.py

```diff
@@ -237,6 +237,13 @@ def setUp(self):
         )
 
     def test_factorySortSetting(self):
+        """
+        Create a core object from an input yaml.
+
+        .. test:: Create core object from input yaml
+            :id: T_ARMI_R_CORE
```

Comment:
I love it.

But looking at the req, I think we need this line somewhere in the test:

```python
self.assertTrue(isinstance(r0.core, Composite))
```

which means we need something like this import at the time:

```python
from armi.reactor.composites import Composite
```